presto-spark-base/src/main/java/com/facebook/presto/spark/execution/common/WorkerNodeProperties.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.common;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * Represents node.properties in Presto context. It helps to create a working baseline for Prestissimo worker.
+ */
+public class WorkerNodeProperties
+        extends Properties
+{
+    private static final String NODE_ENVIRONMENT = "node.environment";
+    private static final String NODE_ID = "node.id";
+    private static final String NODE_LOCATION = "node.location";
+    private static final String NODE_IP = "node.ip";
+    private static final String NODE_MEMORY_GB = "node.memory_gb";
+    private static final String SMC_SERVICE_INVENTORY_TIER = "smc-service-inventory.tier";
```

Comment:
concepts like smc is not even public....

Reply:
Right.. I'll remove non-public configs and let user specify them when needed via setProperty() method